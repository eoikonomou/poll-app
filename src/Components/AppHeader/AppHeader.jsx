import React from 'react';

const AppHeader = () => (
  <div className="header">
    Poll App
  </div>
);

AppHeader.displayName = 'AppHeader';

AppHeader.propTypes = {};

export default AppHeader;
