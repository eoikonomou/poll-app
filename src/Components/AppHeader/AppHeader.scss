.header {
  font-size: 28px;
  font-weight: 900;
  padding: 7px 0;
  border: 1px solid #363636;
  text-align: center;
}
