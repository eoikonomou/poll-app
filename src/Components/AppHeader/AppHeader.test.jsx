import React from 'react';
import { shallow } from 'enzyme';
import AppHeader from './AppHeader';

describe('AppHeader', () => {
  test('should render properly', () => {
    const wrapper = shallow(<AppHeader />);
    expect(wrapper).toMatchSnapshot();
  });
});
