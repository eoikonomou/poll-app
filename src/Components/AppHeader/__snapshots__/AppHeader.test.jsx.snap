// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppHeader should render properly 1`] = `
<div
  className="header"
>
  Poll App
</div>
`;
